src/objective-c/BoringSSL-GRPC.podspec

```diff
@@ -128,22 +128,23 @@ Pod::Spec.new do |s|
   end
   s.subspec 'Implementation' do |ss|
     ss.header_mappings_dir = '.'
-    ss.source_files = 'ssl/*.{h,cc}',
-                      'ssl/**/*.{h,cc}',
-                      '*.{h,c}',
-                      'crypto/*.{h,c}',
-                      'crypto/**/*.{h,c}',
-                      'third_party/fiat/*.{h,c}'
+    ss.source_files = 'ssl/*.{h,c,cc}',
+                      'ssl/**/*.{h,c,cc}',
+                      '*.{h,c,cc}', # for generated file such as err_data.c
```

Comment:
looks like there's only `err_data.c` and `crypto_test_data.cc` and both have been there for long time, so perhaps we can list them explicitly?

Reply:
I'll list err_data.c as the other one is for testing.